stock_shipment_schedule_report/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 {
     "name": "Shipment Schedule Report",
     "summary": "",
-    "version": "10.0.1.0.0",
+    "version": "10.0.2.0.0",
```

Comment:
But there are some changes inside the model file and new field in the wizard model, a server restart will be needed for the changes